Modelica/Electrical/Analog/Lines/Functions/LineGeometry.mo

```diff
@@ -0,0 +1,20 @@
+within Modelica.Electrical.Analog.Lines.Functions;
```

Comment:
Mi idea is that LineCmatrix.mo and LineZmatrix.mo are useful building blocks for future new lines models.
As I mentioned somewhere, the standard approach to model lines in Power Systems is through matrixes Z and C.
Current M_Oline has two flaws:
- it does not allow using resistive parts of off-diagonal elements of Z matrix 
- it uses physical capacitors instead of C matrix for capacitive couplings.
My idea is, when this PR is merged, to create an additional model for lines to overcome these flaws, and using the standard approach of power systems.

Eventually I could get to a model simlar to "Distributed parameter line" in Simulink/Simscape, which extends T_Line for multi-conductor lines: distributed parameter modelling, able to show also travelling waves.

Even without my additional models, in case some power system engineer gets across MSL, they may take advantage of these functions, which contain the most commonly used formulas for simulation of lines. More sophisticated models are bulky, contain parameters very difficult to estimate, and confined to very specialised simulation tools.

Reply:
I only comment the convention used within MSL that a package called "Functions" shall contain only functin classes. I haven't found any other such class containing any other class type. So this shall be consistent within MSL. Either:
- rename the package or
- look into another packages how to resolve this properly.

I think the library officers could hepl here: @christiankral @AHaumer @christophclauss

Reply:
ok.
Let's wait a bit for comments from library officiers on this.

Reply:
and...
I admit that my comment https://github.com/modelica/ModelicaStandardLibrary/pull/4160#discussion_r1248547515 was wordy, without answering your note, just regarding the opportunity to put mo files in a functions folder!

I had just to say that moving TestLineZmatrix and TestLineCmatrix in a folder far from the one containing those functions would make things less clear. 

But if needed because of MSL standards I'll do this.

Reply:
and... (again)
it is not just IEC (and ISO) that use italic for physcical quantities, but, AFAIK, **all technical standardisation bodies**.

The BIPM SI-Brochure-9-EN available from
https://www.bipm.org/en/publications/si-brochure

In that document all physical quantities are written in italic, invariably.

It is the units of measure that must not be italic, even if a lot of writers don't know this (I've found this mistake in many, many papers, even in the most authoritative journals).